precompiles/outposts/osmosis/osmosis.go

```diff
@@ -86,6 +87,19 @@ func NewPrecompile(
 	}, nil
 }
 
+// GetTokenDenom returns the denom associated to the tokenAddress from the
+// erc20 store. Returns an error if the TokenPair associated to the tokenAddress
+// is not found.
+func (p Precompile) GetTokenDenom(ctx sdk.Context, tokenAddress common.Address) (string, error) {
```

Comment:
I think we can move this function as an erc20Keeper method since it is used also in Stride outpost and could be also used in future applications. WDYT?

Reply:
yes, let's do that instead